### DISCUSSION TITLE
Is it possible to persist the text typed or entered in the note box

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#99] [Q&A]
Is it possible to persist the text typed or entered in the note box till another text is typed or send via `texttonote`.
Currently the text in the notebox disappears once a clip action is performed. I would like the notebox contents to persist till we add another notebox content.

**Comment by buggyj** [comment]
That is not possible - It is not a usecase that I thought of....
